strapi/strapi
# enhancement: add history license check

### What does it do?

Doesn't enable the history feature if the license doesn't grant access to it

### Why is it needed?

History is not a free feature

### How to test it?

Use no license. You won't see the content history document action anymore. Opening the history URL directly will trigger an error. Modifying the content will not create history versions.

With a license, the feature should work.
